Source/Layer.js

```diff
@@ -74,60 +192,115 @@ export class Layer extends WrappedObject {
       return frame
     }
 
+    /**
+        Returns a newly created shape, which has been added to this layer,
+        and sets it up using the supplied properties.
+
+        @param properties {dictionary} Properties to apply to the shape.
```

Comment:
Shouldn't this be `@param {dictionary} properties` instead of `@param properties {dictionary}`?